app.py

```diff
@@ -86,7 +85,21 @@ def experience(index=None):
 
     return jsonify({"error": "Method not allowed"}), 405
 
+
+@app.route('/resume/experience/<int:exp_id>', methods=['DELETE'])
```

Comment:
I think we can add this in the existing route for resume/experience. You can just add DELETE as a method there and the implementation